tm2/pkg/sdk/params/keeper.go

```diff
@@ -0,0 +1,157 @@
+package params
+
+import (
+	"log/slog"
+	"strings"
+
+	"github.com/gnolang/gno/tm2/pkg/amino"
+	"github.com/gnolang/gno/tm2/pkg/sdk"
+	"github.com/gnolang/gno/tm2/pkg/store"
+)
+
+const (
+	ModuleName = "params"
+	StoreKey   = ModuleName
+)
+
+type ParamsKeeperI interface {
+	GetString(ctx sdk.Context, key string, ptr *string)
+	GetInt64(ctx sdk.Context, key string, ptr *int64)
+	GetUint64(ctx sdk.Context, key string, ptr *uint64)
+	GetBool(ctx sdk.Context, key string, ptr *bool)
+	GetBytes(ctx sdk.Context, key string, ptr *[]byte)
+
+	SetString(ctx sdk.Context, key string, value string)
+	SetInt64(ctx sdk.Context, key string, value int64)
+	SetUint64(ctx sdk.Context, key string, value uint64)
+	SetBool(ctx sdk.Context, key string, value bool)
+	SetBytes(ctx sdk.Context, key string, value []byte)
+
+	Has(ctx sdk.Context, key string) bool
+	GetRaw(ctx sdk.Context, key string) []byte
```

Comment:
maybe return []byte?

Reply:
It’s already the case. Did you want to suggest something different?